vizro-core/docs/pages/user-guides/dashboard.md

```diff
@@ -203,4 +203,4 @@ title bar. For example, if your Dashboard `title` is "Vizro Demo" and the Page `
 
 Vizro automatically adds [meta tags](https://metatags.io/) to display a preview card when your app is shared on social media and chat
 clients. The preview includes the `URL`, `title`, plus an [image](assets.md/#include-a-meta-tags-image) and
-[Page `description`][vizro.models.Page] (if supplied).  To see an example, try sharing the [Vizro demo app](https://vizro.mckinsey.com/).
+[Page `description`][vizro.models.Page] (if supplied).  To see an example, try sharing an example from the [Vizro examples gallery](https://vizro.mckinsey.com/).
```

Comment:
We might want to have an image when sharing maybe the gallery itself even, but in short term this is fine?